test/datatypes.test.js

```diff
@@ -49,6 +51,12 @@ describe('MySQL specific datatypes', function() {
       assert.ok(mysqlVersion, 'skipping decimal/number test on mysql 5.7');
       return done();
     }
+    if (isWin) {
```

Comment:
Please add a comment telling us why you are skipping decimal number test on mysql 5.7 for windows.